src/components/SquarePaymentForm.tsx

```diff
@@ -200,7 +200,17 @@ export const SquarePaymentForm: React.FC<Props> = (props: Props) => {
     script.onerror = function() {
       onError && onError();
     };
-    document.body.appendChild(script);
+
+    let scriptExists = false;
```

Comment:
What do you think about giving the script tag an id and calling `document.getElementById` instead for this check?

Reply:
Much simpler. Good idea!